query/stdlib/testing/testing.go

```diff
@@ -103,6 +103,14 @@ var FluxEndToEndSkipList = map[string]map[string]string{
 		"join":               "unbounded test",
 		"alignTime":          "unbounded test",
 		"histogram_quantile": "mis-named columns for storage",
+		"distinct":           "failing test",
+		"fill":               "failing test",
+		"histogram":          "failing test",
+		"unique":             "failing test",
+	},
+	"experimental/oee": {
+		"apq":        "failing test",
+		"computeapq": "failing test",
```

Comment:
With this patch, `git diff master -- query/stdlib/testing/testing.go` produces no output